public/sw.js

```diff
@@ -6,7 +6,7 @@
 // cache, then increment the CACHE_VERSION value. It will kick off the ServiceWorker update
 // flow and the old cache will be purged as part of the activate event handler when the
 // updated ServiceWorker is activated
-var CACHE_VERSION = 2
+var CACHE_VERSION = 1
```

Comment:
why not `3`? :)

Reply:
I find I need to change the version during testing to refresh the cache in the browser ...easy to miss change there.

Reply:
I just wondered why you decreased the number when the comment above says to increase it, that’s all :)

Reply:
I can change back in another commit?